### PR TITLE
Issue h attribute

### LINKED ISSUE #14: Attribute definition request for "H"
## Attribute Prefix
Attributes start with H

## Is for an entire prefix or partial implementation 
Full

## If partial, mention segment count
NA

## Definition template

All the attribute definitions are maintained under [definitions](/src/definitions ) directory and here is the template
Template:
```
[
  {
    "display": "<Display Name>",
    "description": "<A very simple description, not going much technical>",
    "link": "<External link to get more details>",
    "propertyName": "<Underneath attribute name>",
    "valuetemplate": "<The template for the value>",
    "categories": ["<A list of categories this attribute can belong>"], 
    "tags": "<A comma separated values for adding tags which is more internal for filter/search etc>",
    "controls": [<List of controls that is required to build the value>]
  }
]
```

### PR DESCRIPTION
**Pull request by atapas**
## Description

Added attributes of h

Fixes # (issue) https://github.com/koustov/css-builder/issues/14

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Verified that added/modified attribues are properly loading on browsing
- [x] Verified that added/modified attribues editing templates are properly rendered
- [x] Verified the output on changes
- [x] Verified the deployment

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas. If applicable.
- [x] I have made corresponding changes to the documentation. If applicable.
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/definitions/h.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            "type": "select",
+            "default": "auto",
+            "label": "Value",
+            "options": [
+              {
+                "name": "Auto",
+                "value": "auto"
+              },
+              {
+                "name": "Set in pixel(px)",
```
Comment (koustov) [inline src/definitions/h.json]: As height can be set as "px" or "%", let the user decide what unit to use along with the value. in another word, there should be an option called "Custom" which will allow the user to enter the value in  <value><unit> format
Refer border-radius

File: src/definitions/h.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                "value": "50px"
+              },
+              {
+                "name": "Set in percentage(%)",
```
Comment (koustov) [inline src/definitions/h.json]: The default value should be derived from the actual attribute's default value which is "auto" for the "height" attribute.

**Review by koustov — CHANGES_REQUESTED** [review]

File: src/definitions/h.json (hunk 1 above)
Comment (atapas) [reply src/definitions/h.json]: So we want it like a select box with values, default(initial), auto. inherit, and custom? When custom selected, we get a range and another selector to select px, %, etc?
Is this kind of relationship possible by framework now?

File: src/definitions/h.json (hunk 2 above)
Comment (atapas) [reply src/definitions/h.json]: @koustov I didn't understand this one fully. I have given the default value as auto for height.

**Review by koustov — APPROVED** [review]
lgtm

**Review by koustov — CHANGES_REQUESTED** [review]
@atapas  please resolve conflicts

**Comment by atapas** [comment]
@koustov DONE!

**Review by koustov — APPROVED** [review]
lgtm

**Pull request merged**